src/index.js

```diff
@@ -1,14 +1,34 @@
-import _ from "lodash";
-import "./style.css";
+import './style.css';
 
-function component() {
-  const element = document.createElement("div");
+import Store from './modules/storage.js';
+import User from './modules/Users.js';
+import UI from './modules/UIclass.js';
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
-  element.classList.add('hello');
+const form = document.querySelector('#form');
 
-  return element;
-}
+// Event : Display UsersScore
+document.addEventListener('DOMcontentLoaded', UI.displayUserScore());
 
-document.body.appendChild(component());
+// Add a user
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  // console.log(e)
+  // get Form values
+  const userName = document.getElementById('username').value;
+  const userScore = document.getElementById('score').value;
+  // console.log(userName, userScore)
+  // instantiate a user
+  const newUser = new User(userName, userScore);
+
+  // Add user to UI
+  UI.addUserToList(newUser);
+  // add user to store
+  Store.addUser(newUser);
+  // console.log(newUser)
+
+  // console.log(newUser)
+  // clear fields
+  UI.clearFields();
+});
+
+// console.log(score);
```

Comment:
- [ ] Please Do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. you can check [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) kindly remove all of them to make your code clean.